product_uom_po_domain/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2021 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Product - Domain on Purchase UoM",
+    "version": "12.0.1.0.1",
+    "category": "Product",
+    "author": "GRAP",
```

Comment:
you should add 'Odoo Community Association (OCA)', this is what is making Travis fail for now.

Reply:
thanks. done.